Tools/TestWebKitAPI/Tests/WTF/CheckedPtr.cpp

```diff
@@ -110,6 +110,7 @@ TEST(WTF_CheckedPtr, Basic)
         EXPECT_EQ(checkedObject.ptrCount(), 2u);
         EXPECT_EQ(ptr1.get(), &checkedObject);
         EXPECT_EQ(ptr2.get(), &checkedObject);
+        IGNORE_CLANG_STATIC_ANALYZER_WARNINGS_ATTRIBUTE("cplusplus.Move")
```

Comment:
If all uses at this time are going to be `"cplusplus.Move"`, then I suggest a specific macro for that one. We can add other specific macros for others that are used repeatedly.

Reply:
The reason all uses are "cplusplus.Move" at this time is that I simply chose that checker to "fix" first.

Okay, I'll add a specific macro for that check.